### PR TITLE
Event detail and edit fixes

### PR DESCRIPTION
**Pull request by botronic**
Some more fixes to the event page:

The lightbox can now be closed by either clicking on the side of the content, or clicking the top right button which has been moved.

There is a field for who is responsible for the event.

General beautifications of the buttons, distances, fonts, etc.

screenshot 2018-12-03 at 13 21 54
screenshot 2018-12-03 at 13 22 03

### CODE REVIEW

File: app/components/Button/styles.css [hunk 1]
```diff
@@ -6,15 +6,20 @@
...
 }
 
 .default {
-  background: #303B4B;
+  background: #303b4b;
   color: white;
 }
 
+.rounded {
+  border-radius: 9999px;
```
Comment (mictab) [inline app/components/Button/styles.css]: Lol wut? I think 100px would be enough and accomplish the same thing :P

File: app/components/EventDetail/index.js [hunk 2]
```diff
@@ -46,58 +47,78 @@ class EventDetail extends Component {
...
           <div>
-            <h4>Location</h4>
+            <h4>
+              <FormattedMessage {...messages.location} />
+            </h4>
             <a
-              href={`https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(
-                event.location
-              )}`}
+              href={`https://www.google.com/maps/search/?api=1&query=
```
Comment (mictab) [inline app/components/EventDetail/index.js]: What happened here? Prettier?

File: app/components/EventEdit/index.js [hunk 3]
```diff
@@ -104,28 +104,40 @@ class EventEdit extends React.Component {
...
-            >
-              <FormattedMessage {...messages.save} />
-            </Button>
-          )}
+          <h2>{event.companyName}</h2>
+          <Button
+            onClick={this.handleSave}
+            type='submit'
+            color={'default'}
+            disabled={!(!saving && !saved)}
```
Comment (mictab) [inline app/components/EventEdit/index.js]: Isn't this the same as `saving || saved`? I think it is.

File: app/components/Lightbox/index.js [hunk 4]
```diff
@@ -1,19 +1,43 @@
-import React, {Component} from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import styles from './styles.css'
+import Button from 'components/Button'
 
 class Lightbox extends Component {
+  constructor(props) {
```
Comment (mictab) [inline app/components/Lightbox/index.js]: Unless we're doing something special in the constructor we don't need to declare it.

File: app/components/Button/styles.css (hunk 1 above)
Comment (botronic) [reply app/components/Button/styles.css]: Yes, assuming we don't have very large buttons, which we probably can do. I will change it.

File: app/components/EventDetail/index.js (hunk 2 above)
Comment (botronic) [reply app/components/EventDetail/index.js]: Nope, the "line length exceeded warning". Should I change it back?

Comment (mictab) [reply app/components/EventDetail/index.js]: Nah, if it works it's fine.

File: app/components/EventEdit/index.js [hunk 5]
```diff
@@ -110,7 +110,7 @@ class EventEdit extends React.Component {
             onClick={this.handleSave}
             type='submit'
             color={'default'}
-            disabled={!(!saving && !saved)}
+            disabled={saving && saved}
```
Comment (mictab) [inline app/components/EventEdit/index.js]: Should be `||`, not `&&`.

Comment (botronic) [reply app/components/EventEdit/index.js]: Wopsie

**Pull request merged**